auth/oauth/error.go

```diff
@@ -60,6 +61,8 @@ const (
 	InvalidRequestObject ErrorCode = "invalid_request_object"
 	// InvalidRequestURI is returned whn the request_uri in the authorization request returns an error or contains invalid data. RFC9101
 	InvalidRequestURI ErrorCode = "invalid_request_uri"
+	// InvalidRequestURIMethod is returned when the request_uri_method is not 'get' or 'post'. OpenID4VP
```

Comment:
sentence not finished

Reply:
I add the source for an error in the description. Now wrapped in () to clarify